application/src/test/java/org/togetherjava/tjbot/commands/tags/TagSystemTest.java

```diff
@@ -0,0 +1,121 @@
+package org.togetherjava.tjbot.commands.tags;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.Tags;
+import org.togetherjava.tjbot.jda.JdaTester;
+
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+final class TagSystemTest {
```

Comment:
Add a DisplayName to all tests in this class

Reply:
This class has clear methods names, I'm confident I know what they do. I'd still rather read a sentence than the methods name though.